corehq/apps/translations/app_translations/upload_form.py

```diff
@@ -112,8 +112,9 @@ def _get_template_translation_el(self):
         default_trans_el = self.itext.find("./{f}translation[@lang='%s']" % default_lang)
         if default_trans_el.exists():
             return default_trans_el
-        all_langs = self.app.langs
-        for lang in all_langs.remove(default_lang):
+        non_default_langs = list(self.app.langs)
```

Comment:
Nitpick: `copy` would make your intent clearer than casting does. I first thought, "What's this about? langs is already a list."

Reply:
makes sense, updated.